### PR TITLE
WIP for volunteer opportunities at location different from org

### PR DESCRIPTION
**Pull request by marianmosley**
https://www.pivotaltracker.com/n/projects/742821/stories/78989204

### CODE REVIEW

File: features/step_definitions/volunteer_op_steps.rb [hunk 1]
```diff
@@ -5,6 +5,20 @@
   expect(current_path).to eq new_organisation_volunteer_op_path org
   fill_in 'Title', :with => title
   fill_in 'Description', :with => desc
+  check 'Use organisation address as location for volunteer opportunity'
+  click_on 'Create a Volunteer Opportunity'
```
Comment (tansaku) [inline features/step_definitions/volunteer_op_steps.rb]: gosh darn - why is hound ignoring our configs!!!

Comment (tansaku) [inline features/step_definitions/volunteer_op_steps.rb]: I want single quotes - I will turn hound off if this persists ...

**Comment by tansaku** [comment]
@marianmosley thanks so much for this - sorry about hound - I'm going to turn him off for the time being I think ...

**Comment by tansaku** [comment]
oh wait - you're not synced to develop so you don't have the latest config file ...

**Comment by tansaku** [comment]
okay so I just ran the merge locally and it's a few files:

```
Auto-merging features/volunteer_opportunities/create_volunteer_opportunities.feature
CONFLICT (content): Merge conflict in features/volunteer_opportunities/create_volunteer_opportunities.feature
Removing features/vcr_cassettes/google/64_pinner_road.yml
Removing features/support/webmock.rb
Auto-merging features/step_definitions/volunteer_op_steps.rb
CONFLICT (content): Merge conflict in features/step_definitions/volunteer_op_steps.rb
Auto-merging features/step_definitions/map_steps.rb
CONFLICT (content): Merge conflict in features/step_definitions/map_steps.rb
Removing doc/models_complete.svg
Removing doc/models_brief.svg
Removing doc/controllers_complete.svg
Removing doc/controllers_brief.svg
Removing doc/README_FOR_APP
Auto-merging db/schema.rb
CONFLICT (content): Merge conflict in db/schema.rb
Auto-merging app/services/map_marker_json.rb
CONFLICT (content): Merge conflict in app/services/map_marker_json.rb
Automatic merge failed; fix conflicts and then commit the result.
```

I just gave them a quick review - they don't look too gnarly ... however there's a lot of work underway related to vol ops.

What's your feeling on this feature?  Something you're keen to work on?  You started working on a different front end refactoring with @johnnymo87 right?   Let me know your plans ...

**Comment by tansaku** [comment]
@marianmosley - any thoughts about how to make progress on this?  A restart with @ZmagoD perhaps?

**Comment by ZmagoD** [comment]
Hi @tansaku I get in contact with @marianmosley and she told me to get ahead with this PR.

**Comment by tansaku** [comment]
@ZmagoD sounds good - should I close this one and you can open a new PR?

**Comment by tansaku** [comment]
@ZmagoD I'll close this and you can open a new one with any code you create

**Comment by ZmagoD** [comment]
Ok @tansaku

**Pull request closed without merging**